scripts/update_wallet_in_factory.js

```diff
@@ -1,4 +1,12 @@
-// Usage: ./execute.sh updateBaseWallet.js staging
+// ///////////////////////////////////////////////////////////////////
+// Script to update the BaseWalet implementation in the WalletFactory contract
+//
+// Can be executed as:
+// ./execute_script.sh update_wallet_in_factory.js <network>
+//
+// where:
+//     - network = [test, staging, prod]
```

Comment:
Can we remove `defaultNetwork = "test"`  in this script to mandate passing the network property